### PR TITLE
False positive - not callable

### PR DESCRIPTION
**Pull request by mbyrnepr2**
## Description
Add a functional test. The issue itself has been fixed in https://github.com/PyCQA/astroid/pull/1311

Refs #5113

### CODE REVIEW

**Comment by DanielNoord** [comment]
@mbyrnepr2 Just because I saw you opened a new PR. What's the state of this PR? And am I correct in assuming that this is targeting #5113?

**Comment by mbyrnepr2** [comment]
> @mbyrnepr2 Just because I saw you opened a new PR. What's the state of this PR? And am I correct in assuming that this is targeting #5113?

Hi @DanielNoord you are right about the issue number. I plan to re-visit it when I have more time; but if you have thoughts on picking it up please do feel free.

**Review by jacobtylerwalls — APPROVED** [review]
Thanks for the updates!

File: tests/functional/n/not_callable.py [hunk 1]
```diff
@@ -201,6 +201,7 @@ def get_number(arg):
 
 get_number(10)()  # [not-callable]
 
+
```
Comment (mbyrnepr2) [inline tests/functional/n/not_callable.py:204]:
```suggestion
```

**Review by Pierre-Sassoulas — APPROVED** [review]

**Pull request merged**